packages/mobile/src/screens/edit-track-screen/fields/AccessAndSaleField/AccessAndSaleField.tsx

```diff
@@ -53,7 +56,11 @@ export const AccessAndSaleField = (props: AccessAndSaleFieldProps) => {
     .map((visibilityKey) => fieldVisibilityLabelMap[visibilityKey])
 
   const trackAvailabilityLabels = useMemo(() => {
-    if ('nft_collection' in (premiumConditions ?? {})) {
+    if (isPremiumContentUSDCPurchaseGated(premiumConditions)) {
+      const amountLabel = `$${premiumConditions.usdc_purchase.price}`
+      return [messages.premium, amountLabel]
```

Comment:
I'm assuming we want to show the $amount to as a label/pill. to confirm with product/design